org.gtk.Gtk3theme.Matcha-dark-azul.json

```diff
@@ -0,0 +1,45 @@
+{
+    "id": "org.gtk.Gtk3theme.Matcha-dark-azul",
+    "branch": "3.36",
+    "runtime": "org.gnome.Sdk",
+    "build-extension": true,
+    "sdk": "org.gnome.Sdk",
+    "runtime-version": "3.36",
+    "appstream-compose": false,
+    "separate-locales": false,
+    "modules": [
+        {
+            "name": "Matcha-dark-azul",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -pm 755 /usr/share/runtime/share/themes/Matcha-dark-azul/gtk-3.0",
+                "cp src/gtk-3.0/gtk-dark-azul.css /usr/share/runtime/share/themes/Matcha-dark-azul/gtk-3.0/gtk.css",
+                "cp src/gtk-3.0/gtk-dark-azul.css /usr/share/runtime/share/themes/Matcha-dark-azul/gtk-3.0/gtk-dark.css",
+                "cp -r src/gtk-3.0/assets-azul /usr/share/runtime/share/themes/Matcha-dark-azul/gtk-3.0/assets",
+                "cp src/gtk-3.0/thumbnail-dark-azul.png /usr/share/runtime/share/themes/Matcha-dark-azul/gtk-3.0/thumbnail.png"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/vinceliuice/Matcha-gtk-theme/archive/2020-05-09.tar.gz",
+                    "sha256": "6c9e4cecb639ad07127ae7561895a92a4796c873210af73baef6832f2a1f37d4"
+                }
+            ]
+        },
+        {
+            "name": "appdata",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p ${FLATPAK_DEST}/share/appdata",
+                "cp org.gtk.Gtk3theme.Matcha-dark-azul.appdata.xml ${FLATPAK_DEST}/share/appdata",
+                "appstream-compose --basename=org.gtk.Gtk3theme.Matcha-dark-azul --prefix=${FLATPAK_DEST} --origin=flatpak org.gtk.Gtk3theme.Matcha-dark-azul"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "org.gtk.Gtk3theme.Matcha-dark-azul.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
I'm not sure what you are talking about. When I read the quickstart, the xml I have seems legit. And when I watch all other themes on flathub, they all have similar appdata. Am I missing something?